Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/defaultPagesSettings/index.jsx

```diff
@@ -90,7 +90,8 @@ class DefaultPagesSettingsPanelBody extends Component {
         let defaultPagesSettings = Object.assign({}, state.defaultPagesSettings);
 
         if (key === "SplashTabId" || key === "HomeTabId" || key === "LoginTabId" || key === "RegisterTabId" ||
-            key === "UserTabId" || key === "SearchTabId" || key === "Custom404TabId" || key === "Custom500TabId") {
+            key === "UserTabId" || key === "SearchTabId" || key === "Custom404TabId" || key === "Custom500TabId" ||
```

Comment:
I'd prefer this to be swapped out to a switch statement due to the number of conditions, however that's my personal preference on coding styles and I don't think it's a requirement to merge if the second reviewer agrees.

Reply:
Just uploaded a switch statement instead